doc/classes/RenderingDevice.xml

```diff
@@ -1868,6 +1868,8 @@
 			Index buffer in 32-bit unsigned integer format. This limits the maximum index that can be specified to [code]4294967295[/code].
 		</constant>
 		<constant name="STORAGE_BUFFER_USAGE_DISPATCH_INDIRECT" value="1" enum="StorageBufferUsage" is_bitfield="true">
+		<constant name="STORAGE_BUFFER_USAGE_READONLY" value="2" enum="StorageBufferUsage" is_bitfield="true">
```

Comment:
```suggestion
		</constant>
		<constant name="STORAGE_BUFFER_USAGE_READONLY" value="2" enum="StorageBufferUsage" is_bitfield="true">
		</constant>
```

Reply:
It took me a little while to understand. @Kimau, you inserted the constant entry inside the other entry, XML-wise.

Reply:
Yep, why I suggested here because this can be cryptic to figure out error wise from the CI

Reply:
Whoops sorry, will fix